scripts/build.ps1

```diff
@@ -233,20 +233,19 @@ function Publish-Package
     Publish-PackageInternal $testHostProject $TPB_TargetFrameworkNS1_4 $testhostNS1_4PackageDir
 
     Write-Log "Package: Publish testhost.x86\testhost.x86.csproj"
-    Publish-PackageInternal $testHostx86Project $TPB_TargetFramework $testhostFullPackageDir
+    Publish-PackageInternal $testHostx86Project $TPB_TargetFramework $testhostFullPackageDir  
 
     # Copy over the Full CLR built testhost package assemblies to the Core CLR and Full CLR package folder.
-    $netFull_Dir = "TestHost"
-    $fullDestDir = Join-Path $coreCLR20PackageDir $netFull_Dir
+    $coreCLRFull_Dir = "TestHost"
+    $fullDestDir = Join-Path $coreCLR20PackageDir $coreCLRFull_Dir
     New-Item -ItemType directory -Path $fullDestDir -Force | Out-Null
     Copy-Item $testhostFullPackageDir\* $fullDestDir -Force -recurse
 
     # Copy over the Full CLR built datacollector package assemblies to the Core CLR package folder along with testhost
     Publish-PackageInternal $dataCollectorProject $TPB_TargetFramework $fullDestDir
-
-    $fullDestDir = Join-Path $fullCLRPackageDir $netFull_Dir
-    New-Item -ItemType directory -Path $fullDestDir -Force | Out-Null
-    Copy-Item $testhostFullPackageDir\* $fullDestDir -Force -recurse
+    
+    New-Item -ItemType directory -Path $fullCLRPackageDir -Force | Out-Null
```

Comment:
$fullCLRPackageDir is already there, no need to create again.

Reply:
Merging this PR in interest of time, will clean this one in next PR.